sdcm/cluster_k8s/__init__.py

```diff
@@ -1689,8 +1689,8 @@ def init_scylla_config_map(self, namespace: str = SCYLLA_NAMESPACE, **kwargs):
         #       So, define some options here by default. It may be extended later if needed.
         with self.remote_scylla_yaml(namespace=namespace) as scylla_yml:
             # Process cluster params
-            if self.params.get("experimental"):
-                scylla_yml["experimental"] = self.params.get("experimental")
+            if self.params.get("experimental_features"):
+                scylla_yml["experimental_features"] = self.params.get("experimental_features")
```

Comment:
we might need to explain that this flag can be overridden by `append_scylla_yaml`, on `experimental_features` configuration doc